### DISCUSSION TITLE
capture Json data filename

### DISCUSSION THREAD

**Discussion by uwegeercken** [duckdb/duckdb#7330] [Q&A]
Hi,

is there a way to capture the path and filename where the json data is originating from? In some circumstances it would be helpfull; e.g. I have a folder hierarchy with a lot of small json files (one object per file) and some files might duplicated. The filename would help to do some cleanup.

Regards,

**Comment by Tishj** [comment]
I believe Laurens is adding support for the `filename` option soon :)
